src/test/java/org/jilt/test/data/defaultvalue/DefaultValueWithLombok.java

```diff
@@ -0,0 +1,39 @@
+package org.jilt.test.data.defaultvalue;
+
+import org.jilt.Builder;
+
+import java.util.Set;
+
+@Builder
+public class DefaultValueWithLombok {
```

Comment:
Let's add `@AllArgsConstructor` here too, instead of writing the constructor by hand.

Reply:
It's fix :)